test/support/postgresql/import_examples.rb

```diff
@@ -17,5 +17,83 @@ def should_support_postgresql_import_functionality
         assert_equal 1, result.num_inserts
       end
     end
+
+    describe "importing objects with ids subobjects" do
```

Comment:
Does this description mean: importing models and their associations? The "ids subobjects" is throwing me for a loop. Not sure my future self will know what we're doing without having to figure it out from the code/examples.